sql/telemetry_derived/bq_main_events/init.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE IF NOT EXISTS
-  `moz-fx-data-shared-prod.telemetry_derived.bq_events` (
+  `moz-fx-data-shared-prod.telemetry_derived.bq_main_events` (
```

Comment:
I had intended that tables in telemetry_derived are for backend use only and should always be versioned. Then the views we present on top of them in `telemetry` can be unversioned. That gives us more wiggle room to completely replace what the backend tables look like while keeping the user-facing view clean and unburdened with knowing what version of the backend table it's hitting.